staging/src/k8s.io/code-generator/cmd/informer-gen/generators/factory.go

```diff
@@ -107,13 +111,29 @@ type sharedInformerFactory struct {
 // NewSharedInformerFactory constructs a new instance of sharedInformerFactory
 func NewSharedInformerFactory(client {{.clientSetInterface|raw}}, defaultResync {{.timeDuration|raw}}) SharedInformerFactory {
   return &sharedInformerFactory{
```

Comment:
I'd recommend having this just call `NewFilteredSharedInformerFactory` with `defaultFilterFunc` as the `filter` arg.